manifests/plugin/remote_execution/ssh.pp

```diff
@@ -0,0 +1,67 @@
+# = Foreman Proxy Remote Execution SSH plugin
```

Comment:
Yea we expect more over time, Salt, Ansible, Mcollective, etc.  I figured keeping it in its own namespace was better, instead of dumping a bunch of `remote_execution_*` files and directories with the other stuff.